cmd/vulcan-github-alerts/main.go

```diff
@@ -113,7 +113,7 @@ You can find more specific information in the resources table for the repository
 )
 
 func main() {
-	run := func(ctx context.Context, target string, optJSON string, state state.State) (err error) {
+	run := func(ctx context.Context, target, assetType, optJSON string, state state.State) (err error) {
```

Comment:
Shouldn't we check if the asset is reachable?

Reply:
Yes, I had some doubts with this one because in the end it's not doing a clone but interacting with the GraphQL API, but yes, in any case we can check reachability/existence of the repo.

I have one question though, because here it seems like we are always making requests to the GHE API with the credentials set in the environment, but for our case, which is performing a git ls-remote on the repo, we should verify that the target host is the same as the GHE host set in the environment no? In order to not pass the credentials to another Github.

Reply:
I added reachability check only passing env credentials if GHE host matches the target one. Tested with an authenticated repo in a GHE.

Reply:
👍